_sass/tiendatek-theme.scss

```diff
@@ -683,6 +692,113 @@ footer {
   font-family: 'MontserratRegular';
 }
 
+// Mobile view
+@include media-breakpoint-down(sm) {
+  .testimonial {
+    font-size: 1em;
+    padding: 1.5em;
+    h1 {
+      margin-bottom: 1em;
+    }
+    iframe {
+      width: 100%;
+    }
+    .img-fluid {
+      max-width: 50%;
+      margin: auto;
+    }
+  }
+
+  footer {
+    font-size: .7em;
+    ul {
+      padding-left: .25em;
+    }
+    .display-3 {
+      font-size: 4em;
+    }
+  }
+
+  .owl-carousel .owl-item img {
+    width: 70%;
+    margin: auto;
+  }
+
+  .intro {
+    font-size: 1em;
+    padding: 1.5em;
+  }
+
+  .intro h2 {
+    font-size: 1.75em;
+    margin-bottom: 1em;
+  }
+
+
+  .tiendatek-cover,
+  .cover-slogan {
+    .display-3 {
+      font-size: 2em;
+      padding: 1em;
+    }
+  }
+
+  #intro .btn-next {
+    margin-top: -30px;
+    padding-bottom: 1.25em;
+    .display-3 {
+      font-size: 3em;
+    }
+  }
+
+  .features-dealers,
+  .features-tt-caja,
+  .features-tt-control {
+    padding: 2em;
+    h2 {
```

Comment:
Why do you leave blank lines only sometimes? IMO, we should leave a blank line between every block in Css...